server/schedule/operator.go

```diff
@@ -79,7 +79,7 @@ func (ap AddPeer) IsFinish(region *core.RegionInfo) bool {
 			log.Warnf("expect %v, but obtain voter %v", ap.String(), p.GetId())
 			return false
 		}
-		return region.GetPendingVoter(p.GetId()) == nil && p.GetId() != ap.PeerID
+		return region.GetPendingVoter(p.GetId()) == nil
```

Comment:
Because `AddNode` won't have any subsequence operators?

Reply:
I think above `if` block already checked if their IDs are matched, so we don't need it here.